content/blog/cleaning-up-your-test-data-builders.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Cleaning Up Your Test Data Builders"
+date: 2019-03-16T13:24:49+01:00
+publishdate: 2019-03-16T13:24:49+01:00
+lastmod: 2019-03-17T00:00:00+01:00
+image: "images/blog/cleaning-up-your-test-data-builders.jpg"
+tags: [ "Cleaner Code", "Test Automation" ]
+comments: true
+draft: true
+---
+
+I still come across a lot of automated tests with many lines of code just to create an object. Even when most data is not relevant for the scenario being tested. Making the tests harder to read and maintain.
+
+Using the [Test Data Builder](http://www.natpryce.com/articles/000714.html) pattern can minimize the lines of code in your test to what's relevant. It's really improved the readability of my test automation code.
```

Comment:
Even kort toelichten, in een paar zinnen, wat het Test Data Builder pattern is. Voor meer informatie doorverwijzen naar de betreffende blog. Op deze manier hoef je het artikel niet te verlaten / een ander artikel eerst moeten lezen voordat je verder kan.